src/topology/continuous_function/bounded.lean

```diff
@@ -1030,4 +1031,82 @@ module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
 
 end normed_algebra
 
+section normed_lattice_ordered_group
+
+variables [topological_space α] [normed_lattice_add_comm_group β]
+
+instance : partial_order (α →ᵇ β) := partial_order.lift (λ f, f.to_fun) (by tidy)
+
+/--
+Continuous normed lattice group valued functions form a meet-semilattice
+-/
+instance : semilattice_inf (α →ᵇ β) :=
+{ inf := λ f g,
+  { to_fun := λ t, f t ⊓ g t,
+    continuous_to_fun := f.continuous.inf g.continuous,
+    bounded' := begin
+      cases f.bounded' with C₁ hf,
+      cases g.bounded' with C₂ hg,
+      refine ⟨C₁ + C₂, λ x y, _⟩,
+      simp_rw normed_group.dist_eq at hf hg ⊢,
+      exact (norm_inf_sub_inf_le_add_norm _ _ _ _).trans (add_le_add (hf _ _) (hg _ _)),
+    end },
+  inf_le_left := λ f g, continuous_map.le_def.mpr (λ _, inf_le_left),
+  inf_le_right := λ f g, continuous_map.le_def.mpr (λ _, inf_le_right),
+  le_inf := λ f g₁ g₂ w₁ w₂, continuous_map.le_def.mpr (λ _, le_inf (continuous_map.le_def.mp w₁ _)
+    (continuous_map.le_def.mp w₂ _)),
+  ..bounded_continuous_function.partial_order }
+
+instance : semilattice_sup (α →ᵇ β) :=
+{ sup := λ f g,
+  { to_fun := λ t, f t ⊔ g t,
+    continuous_to_fun := f.continuous.sup g.continuous,
+    bounded' := begin
+      cases f.bounded' with C₁ hf,
+      cases g.bounded' with C₂ hg,
+      refine ⟨C₁ + C₂, λ x y, _⟩,
+      simp_rw normed_group.dist_eq at hf hg ⊢,
+      exact (norm_sup_sub_sup_le_add_norm _ _ _ _).trans (add_le_add (hf _ _) (hg _ _)),
+    end },
+  le_sup_left := λ f g, continuous_map.le_def.mpr (λ _, le_sup_left),
+  le_sup_right := λ f g, continuous_map.le_def.mpr (λ _, le_sup_right),
+  sup_le := λ f g₁ g₂ w₁ w₂, continuous_map.le_def.mpr (λ _, sup_le (continuous_map.le_def.mp w₁ _)
+    (continuous_map.le_def.mp w₂ _)),
+  ..bounded_continuous_function.partial_order }
+
+instance  : lattice (α →ᵇ β) :=
+{ .. bounded_continuous_function.semilattice_sup, .. bounded_continuous_function.semilattice_inf }
+
+lemma sup_pointwise (f g : α →ᵇ β) (t : α) : (f ⊔ g) t = (f t) ⊔ (g t) := by finish
+
+lemma abs_pointwise (f : α →ᵇ β) (t : α) : |f t| = |f| t :=
+begin
+  unfold has_abs.abs,
+  rw sup_pointwise,
+  simp only [pi.neg_apply, coe_neg],
+end
+
+instance : normed_lattice_add_comm_group (α →ᵇ β) :=
+{ add_le_add_left := begin
+    intros f g h₁ h t,
+    simp only [coe_to_continuous_fun, pi.add_apply, add_le_add_iff_left, coe_add,
+      continuous_map.to_fun_eq_coe],
+    exact h₁ _,
+  end,
+  solid :=
+  begin
+    intros f g h,
+    have i1: ∀ (t:α), ∥f t∥ ≤ ∥g t∥ := begin
+      intro,
+      apply solid,
+      rw [abs_pointwise, abs_pointwise],
+      exact h _,
+    end,
```

Comment:
```suggestion
    have i1: ∀ t, ∥f t∥ ≤ ∥g t∥ := λ t, solid (h t),
```
This suggestions exploits the fact that the proofs of `coe_fn_sup` and `coe_fn_abs` are `rfl`: we don't even need to rewrite!

Reply:
Thanks for the suggestions! I am still very much a beginner.